tests/java_definitions.py

```diff
@@ -0,0 +1,28 @@
+import os
+from pathlib import Path
+
+import jpype
+import jpype.imports
+from jpype.types import *
```

Comment:
I'm ok with this * import as it is self-contained though I still think it'd be cleaner without.

Reply:
Removed this in my last commit!